src/main/java/com/configcat/FetchResponse.java

```diff
@@ -36,22 +37,25 @@ public Object error() {
        return error;
     }
 
-    FetchResponse(Status status, Entry entry, Object error, boolean fetchTimeUpdatable) {
+    public String cfRayId() {return this.cfRayId;}
+
+    FetchResponse(Status status, Entry entry, Object error, boolean fetchTimeUpdatable, String cfRayId) {
         this.status = status;
         this.entry = entry;
         this.error = error;
         this.fetchTimeUpdatable = fetchTimeUpdatable;
+        this.cfRayId = cfRayId;
     }
 
     public static FetchResponse fetched(Entry entry) {
```

Comment:
Shouldn't cfRayId stored in this case as well?

As a matter of fact, this seems to be the only case where `FetchResponse.cfRayId()` is accessed: https://github.com/configcat/java-sdk/blob/b3d995d7553a730991c70c37255b0b68666a1cf3/src/main/java/com/configcat/ConfigFetcher.java#L89

Reply:
Right! My bad, I missed the ! in this line: https://github.com/configcat/java-sdk/blob/b3d995d7553a730991c70c37255b0b68666a1cf3/src/main/java/com/configcat/ConfigFetcher.java#L49   

I will also add to the `fetched` and leave in the other methods in case something changes in the future.